app/src/pages/admin/projectsPage/projectMenu/projectMenu.jsx

```diff
@@ -177,10 +181,11 @@ export class ProjectMenu extends Component {
   navigateToEventsMonitoring = () => {
     const {
       tracking: { trackEvent },
-      project: { organizationSlug, projectSlug },
+      project: { organizationSlug, projectSlug, projectKey, projectName },
     } = this.props;
 
     trackEvent(ADMIN_PROJECTS_PAGE_EVENTS.CLICK_EVENT_MONITORING);
+    this.props.setActiveProjectInfoKey({ projectKey, projectName });
```

Comment:
Does it mean that the `projectInfo` store will hold data related to the one project, but contain the `projectKey, projectName` related to another?
I suggest to avoid such data inconsistencies. It may be achieved just by updating the whole project info.

Reply:
Done.